packages/notifi-react-example/package.json

```diff
@@ -48,5 +48,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "process": "^0.11.10"
```

Comment:
Can you please advise why we need to add `process `as a dev dependency? I believe the webpack file does handle the polyfill error relating to `process`.

Reply:
ah, this was something I noticed while trying to get the card running